ui-tests/cypress/support/pageObjects/administration/fileBrowser/CreateFolderPage.js

```diff
@@ -42,16 +57,52 @@ export default class CreateFolderPage extends Content {
                .eq(0);
   }
 
+  getCreateFolderForm() {
+    return this.getContents()
+               .find(this.createFolderForm);
+  }
+
+  getNameInput() {
+    return this.getCreateFolderForm()
+               .find(this.folderNameInput);
+  }
+
+  getNameInputHelpBlock() {
+    return this.getNameInput()
+               .parent()
+               .siblings(this.nameInputHelpBlock);
+  }
+
+
+  getCancelButton() {
+    return this.getCreateFolderForm()
+               .find(this.cancelButton);
+  }
+
+  clickCancelButton() {
+    this.getCancelButton().click();
+    cy.validateAppBuilderUrlPathname('/file-browser');
```

Comment:
We shouldn't have validation in the page objects...

Reply:
moved validation from page objects